### DISCUSSION TITLE
Submission Nonelectronic Justification (BT-19) & Description (BT-745)

### DISCUSSION THREAD

**Discussion by rengit62** [OP-TED/eForms-SDK#456] [Q&A] [answered]
Hi lab
in doc I read : 
For a Lot, when the electronic submission is prohibited (i.e. the value associated to BT-17 equals "not-allowed"), BT-19 and BT-745 must be specified:

Do you mean this is as a 'sort of advise' or there is a dedicated rule ? 
Because with the following choices, the notice is CVS valid : 
![image](https://github.com/OP-TED/eForms-SDK/assets/52211037/3a0f48f0-bff2-40c9-aaaf-bcaa58ffa9fd)

bye
Renato

**Comment by YvesJo** [comment, marked as answer]
There is a rule that will enforce that. It is however for the time being deactivated and will be reactivated in a next release.
